src/tests/test_utils/common_test_utils/include/common_test_utils/data_utils.hpp

```diff
@@ -223,7 +223,7 @@ void inline fill_data_random(T* pointer,
     const uint32_t k_range = k * range;  // range with respect to k
     random.Generate(k_range);
 
-    if (start_from < 0 && !std::is_signed<T>::value) {
+    if (start_from < 0 && std::is_arithmetic<T>::value && !std::is_signed<T>::value) {
```

Comment:
Why do you add `is_arithmetic` check?

Reply:
**std::is_signed<T>::value == false** for **ov::float16** and **ov::bfloat16**.
So for these 2 types, we don't need to reset start_from to 0